sdk-extensions/incubator/src/main/java/io/opentelemetry/sdk/extension/incubator/fileconfig/LogRecordExporterFactory.java

```diff
@@ -30,12 +32,20 @@ static LogRecordExporterFactory getInstance() {
   @Override
   public LogRecordExporter create(
       LogRecordExporterModel model, SpiHelper spiHelper, List<Closeable> closeables) {
-    OtlpModel otlpModel = model.getOtlp();
-    if (otlpModel != null) {
-      model.getAdditionalProperties().put("otlp", otlpModel);
+    OtlpHttpExporterModel otlpHttpModel = model.getOtlpHttp();
+    if (otlpHttpModel != null) {
+      model.getAdditionalProperties().put("otlp_http", otlpHttpModel);
+    }
+    OtlpGrpcExporterModel otlpGrpcModel = model.getOtlpGrpc();
+    if (otlpGrpcModel != null) {
+      model.getAdditionalProperties().put("otlp_grpc", otlpGrpcModel);
+    }
+    ExperimentalOtlpFileExporterModel otlpFileExporterModel = model.getOtlpFileDevelopment();
+    if (otlpFileExporterModel != null) {
+      model.getAdditionalProperties().put("otlp_file/development", otlpFileExporterModel);
```

Comment:
nit: some of this conditional null/indent thrash can be avoided by using `compute(key, bifunction)`:

```
model.getAdditionalProperties().compute("otlp_http", (k,v) -> model.getOtlpHttp());
...
```
etc.

Reply:
Oh that's an interesting little technique. Quite a bit more succinct! I like it, and applied it everywhere this pattern occurs.